nengo/builder/learning_rules.py

```diff
@@ -36,15 +37,23 @@ class SimBCM(Operator):
         The synaptic weight change to be applied, :math:`\Delta \omega_{ij}`.
     learning_rate : float
         The scalar learning rate, :math:`\kappa`.
+    apply_every : float or None
+        A scalar indicating how often to apply learning rule.
+    n_steps : Signal
+        The integer timestep which the simulator is executing.
```

Comment:
The `n_steps` name suggests to me that this give some total number of steps, but from the docstring it seems it is just the step number? In that case I would just name it `step`, otherwise the docstring should be clarified.